canonical/lxd-ui
# Use release title for displayal #429

## Done

- changed field to display and filter release in image selection modal

Fixes #429 

## QA

1. Run the LXD-UI:
    - On the demo server via the link posted by @webteam-app below. This is only available for PRs created by collaborators of the repo. Ask @lorumic or @edlerd for access.
    - With a local copy of this branch, run as described [here](../HACKING.md#setting-up-for-development).
2. Perform the following QA steps:
    - open the launch image / select image modal
    - check the release column is correctly showing the number and "LTS" for Ubuntu, i.e. "22.04 LTS" and its content is still reasonable for other distros.
